src/app/containers/pleiepenger/Fordeling/Komponenter/GoSysGjelderKategorier.tsx

```diff
@@ -0,0 +1,79 @@
+import {IFordelingState} from 'app/models/types';
+import {
+    hentGjelderKategorierFraGosys
+} from 'app/state/actions';
+import {v4 as uuidv4} from 'uuid';
+import {RootStateType} from 'app/state/RootState';
+import intlHelper from 'app/utils/intlUtils';
+import {Select} from 'nav-frontend-skjema';
+import React, {useEffect, useState} from 'react';
+import {injectIntl, WrappedComponentProps,} from 'react-intl';
+import {connect} from 'react-redux';
+import VerticalSpacer from "../../../../components/VerticalSpacer";
+
+export interface IOwnProps {
+    fordelingState: IFordelingState;
+    hentGjelderKategorier: typeof hentGjelderKategorierFraGosys;
+    setGosysKategoriJournalforing: (gosysKategori: string) => void;
+}
+
+type IGosysGjelderKategorierProps = WrappedComponentProps & IOwnProps;
+
+const GosysGjelderKategorierComponent: React.FunctionComponent<IGosysGjelderKategorierProps> = (
+    props: IGosysGjelderKategorierProps
+) => {
+    const {
+        intl,
+        fordelingState,
+        hentGjelderKategorier,
+        setGosysKategoriJournalforing
+    } = props;
+
+    const [valgtKategori, setValgtKategori] = useState<string>('');
+
+    useEffect(() => {
+        hentGjelderKategorier();
+    }, [])
+
+    const harKategorierBlivitHentet = typeof fordelingState.isAwaitingGosysGjelderResponse !== 'undefined'
+        && !fordelingState.isAwaitingGosysGjelderResponse
+        && typeof fordelingState.isAwaitingGosysGjelderResponse !== 'undefined'
```

Comment:
Samme sjekk som i linje 38

Reply:
Kan man kutte linje 38 - 40 her til `fordelingState.isAwaitingGosysGjelderResponse === false`?

Reply:
Ajdå! Har fjernet den nå. 

Dersom jeg försöker accessera fordeling.isAwaitingGosysGjelderResponse og den er undefined så borde jeg få ReferenceError i console väl? Att isAwaitingGosysGjelderResponse is not defined. Därav sjekk om isAwaitingGosysGjelderResponse innan jeg försöker accessera objektet

Reply:
Ser dock inte ut som att det blir referenceError nå når jeg tester så kan fjerne typeof sjekken :-) Skulle det bli problem i Q kan jag bara lägga till den igen